### PR TITLE
Fix Python example code

### PR DESCRIPTION
**Pull request by 2Cubed**
It... it works! Huzzah!

### CODE REVIEW

File: python_api.rst [hunk 1]
```diff
@@ -12,17 +12,30 @@ Quick Example Code
...
+    path = "https://beam.pro/api/v1"
+    auth = {
+        "username": "USERNAME",
+        "password": "PASSWORD"
+    }
+
+    session = Session()
+    auth['id'] = session.post(path + "/users/login", auth).json()['channel']['id']
+
+    data = session.get(path + "/tetris/{id}/robot".format(id=auth['id'])).json()
```
Comment (connor4312) [inline python_api.rst]: todo for myself is to add this into the Python client library. Can you put this into a login() function or something for now though? Just to keep things a little cleaner.

Comment (2Cubed) [inline python_api.rst]: @connor4312 Of course!

**Comment by 2Cubed** [comment]
@connor4312 Functionified.

**Comment by connor4312** [comment]
Thanks!

**Pull request merged**